global_path:
  n_points_clothoid: 400
  n_points_spline: 100
  epsilon: 0.3
  min_wapoints_distance: 8.0
  frame: map
  x: [0, 80, 100]
  y: [-2, -2, -2]
  theta: [0, 0, 0]
  #x: [5.999974, 6.000977, 5.999648, 5.998551, 5.998929, 6.001147, 6.000591, 5.999835, 5.999424, 5.992386, 5.960005, 5.894092, 5.785831, 5.662384, 5.474763, 5.257790, 4.954222, 4.653134, 4.270108, 3.920702, 3.486520, 3.108429, 2.598239, 2.128317, 1.473353, 0.843232, -0.012317, -0.785626, -1.803734, -2.691600, -3.835231, -4.804266, -5.938567, -6.802780, -7.747909, -8.426603, -9.170494, -9.699457, -10.276966, -10.813954, -11.444327, -12.090203, -12.855695, -13.563038, -14.322385, -15.171259, -15.941659, -16.646634, -17.331804, -17.918581, -18.462962, -18.934280, -19.389638, -19.795445, -20.309152, -20.782267, -21.298785]
  #y: [-16.278403, -16.278186, -16.278476, -16.276690, -16.280219, -16.280961, -16.279365, -16.277076, -16.275668, -16.255332, -16.174460, -16.025529, -15.755009, -15.434709, -14.946130, -14.396587, -13.608882, -12.792152, -11.646027, -10.542251, -9.059796, -7.691001, -5.899521, -4.309098, -2.280286, -0.564660, 1.512921, 3.184297, 5.197224, 6.834744, 8.817635, 10.462699, 12.442570, 14.084165, 16.080787, 17.712481, 19.709892, 21.321532, 23.174560, 24.872687, 26.634347, 28.313148, 30.056338, 31.490299, 32.922974, 34.436723, 35.873758, 37.232642, 38.693794, 40.129694, 41.589987, 42.983207, 44.430220, 45.762326, 47.265660, 48.458183, 49.716408]
  #theta: [1.948152, 1.948144, 1.948160, 1.948194, 1.948128, 1.948098, 1.948171, 1.948090, 1.947916, 1.947885, 1.947660, 1.948994, 1.949126, 1.949434, 1.947867, 1.943497, 1.930650, 1.913042, 1.887984, 1.867460, 1.845484, 1.840516, 1.850215, 1.865938, 1.900329, 1.937780, 1.982449, 2.017111, 2.053890, 2.081107, 2.103719, 2.104422, 2.077221, 2.041071, 1.992794, 1.952766, 1.908574, 1.880500, 1.871441, 1.890066, 1.921798, 1.963961, 2.012747, 2.049987, 2.076314, 2.083546, 2.063287, 2.028775, 1.988814, 1.950002, 1.913035, 1.879521, 1.859162, 1.876571, 1.915143, 1.951931, 1.977990]
